File: practica2_820574_839304/java/Pila_ints.java
package practica2_820574_839304.java;
import java.util.Stack;

public class Pila_ints {
    Stack<Integer> pila;

    Pila_ints(){
        pila = new Stack<Integer>();
    }

    public void insertar(int n){
        pila.push(n);
    }

    public int extraer(){
        return pila.pop();
    }

    public int cima(){
        return pila.peek();
    }
}

File: practica2_820574_839304/c++/pila_ints.cc
/*
* Fichero: pila_ints.cc
* Autores: Alicia Lazaro Huerta y Manel Jorda Puig Rubio
* Fecha: 27/02/2023
*/

#include "pila_ints.h"

PilaInts::PilaInts(){}

void PilaInts::insertar(int a)
{
    pila.push(a);
}
        
void PilaInts::extraer()
{
    pila.pop();
}

int PilaInts::cima()
{
    return pila.top();
}
